fw/http_tbl.h

```diff
@@ -23,6 +23,25 @@
 
 #include "http.h"
 
+typedef enum {
+    TFW_HTTP_RES_VHOST,
+    TFW_HTTP_RES_REDIR,
+    _TFW_HTTP_RES_COUNT
+} tfw_http_res_act_t;
+
+typedef struct {
+    char *url;
+    unsigned int resp_code;
```

Comment:
`unsigned short` is enough. The structure should be extended - see other comments, but also remember to not have memory holes.

Reply:
Parsing wants integers. Making an intermediate variable looks redundant